docs/modules/ROOT/pages/mgmt_services/security_group_manager.adoc

```diff
@@ -0,0 +1,1132 @@
+			"name": null,
+			"description": null,
+			"tenant_id": "3d53801c-32ce-4e97-9572-bb966fab628",
+			"security_group_id": "98d5b83c-2aed-4bc7-aa22-303960fd0017",
+			"remote_group_id": null,
+			"direction": "ingress",
+			"remote_ip_prefix": null,
+			"protocol": null,
+			"port_range_max": null,
+			"port_range_min": null,
+			"ether_type": "IPv6"
+		}],
+		"create_at": "2020-05-26 16:06:14",
+		"update_at": "2020-05-26 16:06:14"
+	}
+}, {
+	"security_group": {
+		"project_id": "3d53801c-32ce-4e97-9572-bb966fab628",
+		"id": "3d53801c-32ce-4e97-9572-bb966f476ec",
+		"name": "securityGroup2",
+		"description": "This is another security group",
+		"tenant_id": "3d53801c-32ce-4e97-9572-bb966fab628",
+		"security_group_rules": [{
+			"project_id": null,
+			"id": "bb71feb1-abd2-403f-93ca-7d87fe3e6082",
+			"name": null,
+			"description": null,
+			"tenant_id": "3d53801c-32ce-4e97-9572-bb966fab628",
+			"security_group_id": "3d53801c-32ce-4e97-9572-bb966f476ec",
+			"remote_group_id": null,
+			"direction": "egress",
+			"remote_ip_prefix": null,
+			"protocol": null,
+			"port_range_max": null,
+			"port_range_min": null,
+			"ether_type": "IPv4"
+		}, {
+			"project_id": null,
+			"id": "50ec33df-fdc2-4256-9378-96b471b544d3",
+			"name": null,
+			"description": null,
+			"tenant_id": "3d53801c-32ce-4e97-9572-bb966fab628",
+			"security_group_id": "3d53801c-32ce-4e97-9572-bb966f476ec",
+			"remote_group_id": null,
+			"direction": "egress",
+			"remote_ip_prefix": null,
+			"protocol": null,
+			"port_range_max": null,
+			"port_range_min": null,
+			"ether_type": "IPv6"
+		}],
+		"create_at": "2020-05-26 16:06:14",
+		"update_at": "2020-05-26 16:06:15"
+	}
+}]
+....
+anchor:Create_Security_Group_Rule[]
+**(6) Create Security Group Rule**
+
+* Method: `POST`
+
+* Request: `/project/{project_id}/security-group-rules`
+
+* Request Parameter: `@PathVariable String projectId`
+
+* Response: `Security Group Rule `
+
+* Normal response codes: 200
+
+* Error response codes: 400, 412, 500
+
+* Example
+....
+Request:
+http://127.0.0.1:8080/project/3d53801c-32ce-4e97-9572-bb966fab628/security-group-rules
+
+Body:
+{
+	"security_group_rule": {
+		"project_id": "3d53801c-32ce-4e97-9572-bb966fab628",
+		"id": "3d53801c-32ce-4e97-9572-bb966fa6b23",
+		"name": null,
+		"description": null,
+		"tenant_id": "3d53801c-32ce-4e97-9572-bb966fab628",
+		"security_group_id": "3d53801c-32ce-4e97-9572-bb966f476ec",
+		"remote_group_id": null,
+		"direction": "ingress",
+		"remote_ip_prefix": null,
+		"protocol": "tcp",
+		"port_range_max": 65535,
+		"port_range_min": 1,
+		"ether_type": "IPv4"
+	}
+}
+
+Response:
+{
+	"security_group_rule": {
+		"project_id": "3d53801c-32ce-4e97-9572-bb966fab628",
+		"id": "3d53801c-32ce-4e97-9572-bb966fa6b23",
+		"name": null,
+		"description": null,
+		"tenant_id": "3d53801c-32ce-4e97-9572-bb966fab628",
+		"security_group_id": "3d53801c-32ce-4e97-9572-bb966f476ec",
+		"remote_group_id": null,
```

Comment:
remote_group_id and remote_group_prefix can't be null at the same time. Please check if we have appropriate check in the code.

Reply:
yes, we have.

Reply:
In this case, can we use update the sample response? We have both fields null here.